src/tests/bitsets/test_stdlib_bitset_large.f90

```diff
@@ -0,0 +1,1476 @@
+            form='formatted', action='write' )
+        call set12 % write_bitset(unit)
+        call set11 % write_bitset(unit)
+        call set10 % write_bitset(unit)
+        close( unit )
+        open( newunit=unit, file='test.txt', status='old', &
+            form='formatted', action='read' )
+        call set13 % read_bitset(unit)
+        call set15 % read_bitset(unit)
+        call set14 % read_bitset(unit)
+        if ( set14 /= set10 .or. set15 /= set11 .or. set3 /= set12 ) then
+            error stop procedure // ' transfer to and from units using ' // &
+                ' bitset literals for bits > 64 failed.'
+        else
+            write(*,*) 'Transfer  bits > 64 to and from units using ' // &
+                'plain write_bitset_unit and read_bitset_unit succeeded.'
+        end if
+
+        close( unit )
+
+        open( newunit=unit, file='test.txt', status='replace', &
+            form='formatted', action='write' )
+        call set2 % write_bitset(unit, advance='no')
+        call set1 % write_bitset(unit, advance='no')
+        call set0 % write_bitset(unit)
+        close( unit )
+        open( newunit=unit, file='test.txt', status='old', &
+            form='formatted', action='read' )
+        call set3 % read_bitset(unit, advance='no')
+        call set4 % read_bitset(unit, advance='no')
+        call set5 % read_bitset(unit)
+        if ( set5 /= set0 .or. set4 /= set1 .or. set3 /= set2 ) then
+            error stop procedure // ' transfer to and from units using ' // &
+                ' bitset literals with advance == "no" failed.'
+        else
+            write(*,*) 'Transfer to and from units using ' // &
+                'write_bitset_unit and read_bitset_unit with ' // &
+                'advance=="no" succeeded.'
+        end if
+
+        close( unit )
+
+        open( newunit=unit, file='test.txt', status='replace', &
+            form='formatted', action='write' )
+        call set12 % write_bitset(unit, advance='no')
+        call set11 % write_bitset(unit, advance='no')
+        call set10 % write_bitset(unit)
+        close( unit )
+        open( newunit=unit, file='test.txt', status='old', &
+            form='formatted', action='read' )
+        call set13 % read_bitset(unit, advance='no')
+        call set14 % read_bitset(unit, advance='no')
+        call set15 % read_bitset(unit)
+        if ( set15 /= set10 .or. set14 /= set11 .or. set13 /= set12 ) then
+            error stop procedure // ' transfer to and from units using ' // &
+                ' bitset literals for bitss > 64 with advance == "no" failed.'
+        else
+            write(*,*) 'Transfer bits > 64 to and from units using ' // &
+                'write_bitset_unit and read_bitset_unit with ' // &
+                'advance=="no" succeeded.'
+        end if
+
+        open( newunit=unit, file='test.bin', status='replace', &
+            form='unformatted', action='write' )
```

Comment:
It would be nice to have the same tests with `access=stream`

Reply:
Done

Reply:
Thank you.